specs/v1.yaml

```diff
@@ -1257,6 +1441,141 @@ components:
           example: direct
         address:
           $ref: "#/components/schemas/GrantAddress"
+    RecurringGrant:
+      type: object
+      description: |-
+        A RecurringGrant represents a successfully initiated recurring grant request with a Donor Advised Fund. RecurringGrants are created when a person interacts with an instance of Chariot Connect and successfully submits a recurring grant and completes the workflow. There can be many recurring grants associated with a Connect object and therefore a Nonprofit.
+      required:
+        - id
+        - workflowSessionId
+        - fundId
+        - amount
+        - startDate
+      properties:
+        id:
+          type: string
+          readOnly: true
+          description: The unique identifier for the object
+          format: uuid
+          example: cfe09e64-6a74-4dab-a565-361185a6f248
+        userFriendlyId:
+          type: string
+          readOnly: true
+          description: Often referred to as the "Chariot ID", this is the ID that will be included in the payment from the DAF provider.
+          example: chariot-1234455
+          deprecated: true
+        trackingId:
+          type: string
+          readOnly: true
+          description: The tracking ID for the grant
+          example: L9E182VBGP
+        workflowSessionId:
+          type: string
+          readOnly: true
+          description: ID of the Connect Workflow Session associated with this grant
+          format: uuid
+          example: 2d4b2a43-a5b4-4be1-ad1f-f932016ca4a6
+        startDate:
```

Comment:
it might be useful to align on the properties of this object:
- startDate - not entirely sure this is necessary but curious to hear rationale
- endDate - this is what I would call the "terminated_at` date
- status - not sure this is necessary right now
- feeDetail - also probably can be removed
- statuses & payment channel can be removed

Reply:
This works for me.